diesel_tests/build.rs

```diff
@@ -39,7 +39,7 @@ mod outer {
         let database_url = ::std::env::var("DATABASE_URL")
             .expect("DATABASE_URL must be set to run tests");
         let connection = PgConnection::establish(&database_url).unwrap();
-        let migrations_dir = migrations::find_migrations_directory().unwrap().join("postgresql");
+        let migrations_dir = migrations::find_migrations_directory().unwrap().join("../../migrations/postgresql");
```

Comment:
Actually, can you move the migrations that you added in this PR to a directory name other than `migrations` so this doesn't need to change?